Autopilot/PathManager/Src/pathStateClasses.cpp

```diff
@@ -1,11 +1,42 @@
 #include "pathStateClasses.hpp"
 
+/***********************************************************************************************************************
+ * Static Member Variable Declarations
+ **********************************************************************************************************************/
+
 bool isError; 
```

Comment:
If this var is file scope, it should declared a static. Otherwise it would be global and other files may have access to it